code/obj/item/table_rack_parts.dm

```diff
@@ -222,6 +222,20 @@ RACK PARTS
 	furniture_type = /obj/table/reinforced/chemistry/auto
 	furniture_name = "chemistry countertop"
 
+//bookshelf part construction
+	attackby(obj/item/W as obj, mob/user as mob)
```

Comment:
Wait, you get bookshelves by reinforcing chemistry table parts? How does that make any sense?

Reply:
ah shit thanks for catching that, i copy pasted in the wrong part! let me revise that quick!

Reply:
just pushed an update to fix this